.github/scripts/print-gradle-workers-max.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# We know the gradle daemon takes 1GiB (see gradle.properties)
+DAEMON_BYTES=1073741824
+
+# We'll leave some buffer space for other system resources / processes
+OTHER_BYTES=2147483648
+
+TOTAL_SYSTEM_BYTES="$(free --bytes | grep Mem | awk -F " " '{print $2}')"
```

Comment:
Where does `free` come from? I don't have it locally as a shell built-in or in my path.

Reply:
It's a very common Linux command line utility.

https://man7.org/linux/man-pages/man1/free.1.html

https://gitlab.com/procps-ng/procps/-/blob/master/src/free.c